examples_tests/51.RadixSort/main.cpp

```diff
@@ -88,71 +99,79 @@ static void DebugCompareGPUvsCPU(smart_refctd_ptr<IGPUBuffer> gpu_buffer, T* cpu
 	}
 }
 
-static void ExclusiveSumScan(IVideoDriver* driver, smart_refctd_ptr<IGPUBuffer> in_gpu, const uint32_t in_count, IGPUDescriptorSet* ds_upsweep,
-	IGPUComputePipeline* upsweep_pipeline, IGPUDescriptorSet* ds_downsweep, IGPUComputePipeline* downsweep_pipeline)
+static void RadixSort(IVideoDriver* driver, const SBufferRange<IGPUBuffer>& in_gpu_range,
+	IGPUDescriptorSet* ds_histogram, IGPUComputePipeline* histogram_pipeline,
+	IGPUDescriptorSet*  ds_scan, IGPUComputePipeline*  upsweep_pipeline, IGPUComputePipeline* downsweep_pipeline,
+	IGPUDescriptorSet* ds_scatter, IGPUComputePipeline* scatter_pipeline)
```

Comment:
this looks like something that should be in the RadixSort class (or at least large parts of it)